fabric-renderer-registries-v1/src/main/java/net/fabricmc/fabric/api/client/rendereregistry/v1/ItemOverlayRenderer.java

```diff
@@ -0,0 +1,94 @@
+package net.fabricmc.fabric.api.client.rendereregistry.v1;
+
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.network.ClientPlayerEntity;
+import net.minecraft.client.render.BufferBuilder;
+import net.minecraft.client.render.Tessellator;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.VertexFormats;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.util.math.Matrix4f;
+import org.lwjgl.opengl.GL11;
+
+public interface ItemOverlayRenderer {
+	/**
+	 * @return <code>true</code> to cancel vanilla overlay rendering.
```

Comment:
Is this rendered before or after vanilla? Docs should specify that

Reply:
It's before, I'll go add that